cmd/git-sync/main.go

```diff
@@ -687,10 +687,16 @@ func setupGitCookieFile() error {
 		return fmt.Errorf("error: could not access git cookie file: %v", err)
 	}
 
-	cmd := exec.Command(*flGitCmd, "config", "--global", "http.cookiefile", pathToCookieFile)
-	output, err := cmd.CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("error configuring git cookie file %v: %s", err, string(output))
+	if _, err = runCommand(
```

Comment:
Can you please linewrap this as:

```
if _, err = runCommand(context.TODO(), "",
        *flGitCmd, "config", "--global", "http.cookiefile", pathToCookieFile); err != nil {
```

or just not bother to linewrap it? :)

Reply:
Done